release_notes/46-release-notes.adoc

```diff
@@ -108,6 +126,39 @@ include::snippets/technology-preview.adoc[]
 
 With this release, {product-title-short} adds the ability to manage {product-title-short} policies as Kubernetes custom resources, enabling GitOps workflows such as Argo CD. For more information, see xref:../operating/manage_security_policies/custom-security-policies.adoc#policy-as-code-about_custom-security-policies[Managing policies as code].
 
+[id="policy-violation-views_{context}"]
+=== Policy violation views
+
+* {product-title-short} 4.6 introduces a new drop-down selector in the policy violations view. You can use it to focus on violations related to user application workloads, platform workloads, or both.
+* The policy violations view lists violations across three panels to easily distinguish between active, resolved, and attempted violations.
+
+[id="upgraded-view-filters_{context}"]
+=== Upgraded view filters
+
+The violations view now includes comprehensive filtering capabilities similar to those introduced earlier in *Vulnerability Management*:
+
+* Filter by cluster, namespace, and deployment.
+* Filter by Violation date, time, and entity type.
+* Filter by Policy name, category, severity, and other options.
+
+[id="policy-editor-updates_{context}"]
+=== Policy editor updates
+
+* The policy editor now includes reordered sections to reflect the logical policy structure better, splitting the policy definition part (rules and metadata) from the behavior part (scope and actions).
+* The editor recognizes *Policy as Code* capabilities and shows a warning when editing an *Externally Managed* policy.
+
+[id="exposure-analysis-network-policy_{context}"]
+=== Exposure analysis in network policy generator
+
+When you create a connectivity map by using the build-time network policy generator, an option is now available to show open connections. This configuration with open connections increases the exposure risk. Therefore, such connections are a good candidate for limiting exposure. For example, a network policy might allow ingress to a pod from any namespace instead of specifying individual consumers.
+
+To include exposure information, use the `--exposure` option:
+
+[source,terminal]
+----
+$ roxctl netpol connectivity map --exposure
+----
```

Comment:
Is this feature documented in our docs that could serve as a reference here?
if yes, we could remove this piece.